src/plugin/places/placesmanager.js

```diff
@@ -162,6 +165,44 @@ plugin.places.PlacesManager.prototype.getPlacesRoot = function() {
 };
 
 
+/**
+ * Iterate over all nodes in the places tree and add all folders to the list.
+ * @param {os.structs.ITreeNode=} opt_node
+ * @private
+ */
+plugin.places.PlacesManager.prototype.addFolders_ = function(opt_node) {
+  var node;
+  if (!opt_node) {
+    node = this.placesRoot_;
+  } else {
+    node = opt_node;
+  }
+  if (node) {
+    if (node.isFolder() && !this.folders_.includes(node)) {
```

Comment:
I don't believe it is possible to parent a folder under multiple parent folders, and thus the `includes` check should never be necessary.

Reply:
Fixed